e2e/cypress/integration/messaging/inline_markdown_image_link_open_link_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let testChannel;
+    let testUser;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testUser = user;
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiLogin(testUser);
+
+            // # Visit a test channel and post a message
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+        });
```

Comment:
Looks like declared variables are not used elsewhere. Also login to test user if no other admin function is needed after init, can simply use `loginAfter: true`.
```suggestion
        cy.apiInitSetup({loginAfter: true}).then(({team, channel, user}) => {
            // # Visit a test channel and post a message
            cy.visit(`/${team.name}/channels/${channel.name}`);
        });
```